### ISSUE TITLE
Decoder giving error '-' used multiple times for input.

### ISSUE THREAD

**Issue by ronilp** [gouwsmeister/TextCleanser#3]
There seems to be an error with this line - 

`command = "%slattice-tool -in-lattice - -read-mesh -posterior-decode -zeroprob-word blemish -use-server %s " % (LATTICE_TOOL_DIR, NGRAM_SERVER_IP)`

**Comment by wanghm92** [comment]
Hello! I'm facing similar problem, where the decoder error output is always '-'. Please refer to Issue#4 for details. May I ask whether your problem has been solved? Thanks!

**Comment by ronilp** [comment]
No, I am still facing the issue. Please let me know if you are able to solve this.

**Comment by wanghm92** [comment]
What do you see if you run the "./start_ngram_server.sh"?

**Comment by wanghm92** [comment]
Cost me a whole day... Use 1.5.12/1.6.0 version of SRILM will do.

**Comment by ygorg** [comment]
The problem might be in SRILM >1.6.0 in `srilm/lattice/src/lattice-tool.cc` the file is opened twice which is in case of stdin not possible (cf. processLattice function).
